cobc/codegen.c

```diff
@@ -2567,6 +2567,22 @@ output_literals_figuratives_and_constants (void)
 
 /* Collating tables */
 
+#include "cconv.h"
+
+/* Outputs identity conversion if `table == NULL`. */
```

Comment:
please adjust that, something like "from specified table or, when omitted as native output"